src/starcheck.pl

```diff
@@ -656,6 +679,27 @@ sub json_obsids{
     $out .= qq{${font_stop}\n};
 }
 
+
+# Run independent attitude checker
+my $CHAR_REQUIRED_AFTER = '2015:315:00:00:00.000';
+if ((defined $char_file) or ($bs[0]->{time} > date2time($CHAR_REQUIRED_AFTER))){
+    $out .= "------------  VERIFY TARGETS (SI_ALIGN CHECK)  -----------------\n\n";
```

Comment:
This is actually verify PCAD attitudes based on targets, though we do the computation in the other direction.